build/builtin_actors_gen.go

```diff
@@ -176,10 +170,9 @@ var EmbeddedBuiltinActorsMetadata []*BuiltinActorsMetadata = []*BuiltinActorsMet
 		"verifiedregistry": MustParseCid("bafk2bzacebzndvdqtdck2y35smcxezldgh6nm6rbkj3g3fmiknsgg2uah235y"),
 	},
 }, {
-	Network:      "caterpillarnet",
-	Version:      10,
-	BundleGitTag: "dev/20230207-pre-rc.7",
-	ManifestCid:  MustParseCid("bafy2bzaceajatwdn4jlfiweb2cuwqtnrdwdxlju2vvejlqsytocnc76xnwm2g"),
+	Network:     "caterpillarnet",
+	Version:     10,
+	ManifestCid: MustParseCid("bafy2bzaceajatwdn4jlfiweb2cuwqtnrdwdxlju2vvejlqsytocnc76xnwm2g"),
```

Comment:
Dropping all of these bundlegittags until https://github.com/filecoin-project/lotus/issues/10185#issuecomment-1422864836 is resolved